README.adoc

```diff
@@ -35,27 +35,37 @@ Minishift.
 
 === What is Origin Community Distribution of Kubernetes (OKD)?
 
-OKD, formerly known as OpenShift Origin, is the upstream open source project for all OpenShift products. OKD is a Kubernetes-based platform with
-added functionality. OKD streamlines the DevOps process by providing an intuitive development pipeline. It also provides integration with multiple tools 
-to make the deployment and management of cloud applications easier.
+OKD, formerly known as OpenShift Origin, is the upstream open source project for all OpenShift products. 
+OKD is a Kubernetes-based platform with added functionality. 
+OKD streamlines the DevOps process by providing an intuitive development pipeline. 
+It also provides integration with multiple tools to make the deployment and management of cloud applications easier.
 
-To learn more about OKD, check out the https://www.okd.io/[official OKD page^]. To learn more about the different platforms that Red Hat OpenShift offers, 
-check out the https://docs.openshift.com[official OpenShift documentation^]. If you would like to learn more about Kubernetes, check out 
+To learn more about OKD, check out the https://www.okd.io/[official OKD page^]. 
+To learn more about the different platforms that Red Hat OpenShift offers, 
+check out the https://docs.openshift.com[official OpenShift documentation^]. 
+If you would like to learn more about Kubernetes, check out 
 the https://openliberty.io/guides/kubernetes-intro.html[Deploying microservices to Kubernetes^] guide.
 
 Using Maven, you will build the `system` microservice that collects basic system properties from your system and the `inventory` microservice that will 
 interact with the `system` microservice. Then, you will learn how to deploy both to the cluster and establish communication between them.
 
-You will use Minishift, a tool for you to run OKD on a local system. Minishift allows developers to deploy a quick and easy OKD cluster for application development.
+You will use Minishift, a tool for you to run OKD on a local system. 
+Minishift allows developers to deploy a quick and easy OKD cluster for application development.
+
+Minishift is based on OKD 3.11.
+To run OKD 4.1 or newer on your local system, you can instead use the CodeReady Containers tool.
```

Comment:
To run OKD 4.1 or newer on your local system, you can instead use the CodeReady Containers tool.
--->
To run OKD 4.1 or newer on your local system, you can use the CodeReady Containers tool instead.